### PR TITLE
Adapter yang-sprotty to latest sprotty, uplift version to 0.2.0

### PR DESCRIPTION
**Pull request by huyuwen**

### CODE REVIEW

File: src/di.config.ts [hunk 1]
```diff
@@ -9,22 +9,24 @@ import { ConsoleLogger, ExpandButtonHandler, ExpandButtonView, HtmlRoot,
...
 import { YangModelFactory } from "./model-factory";
+import 'sprotty/css/sprotty.css'
+import 'sprotty-theia/css/theia-sprotty.css'
+import "../css/dark/diagram.css"
 
 const yangDiagramModule = new ContainerModule((bind, unbind, isBound, rebind) => {
     rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope()
     rebind(TYPES.LogLevel).toConstantValue(LogLevel.warn)
     rebind(TYPES.IModelFactory).to(YangModelFactory).inSingletonScope()
-    bind(TYPES.PopupModelFactory).toConstantValue(popupModelFactory)
```
Comment (spoenemann) [inline src/di.config.ts]: The `popupModelFactory` should be refactored to fit the current API.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "yang-sprotty",
-  "version": "0.1.9",
+  "version": "0.2.0",
```
Comment (spoenemann) [inline package.json]: The version should be lifted up when we do the next release, or what do you think, @JanKoehnlein?

Comment (JanKoehnlein) [reply package.json]: Better now.

**Comment by JanKoehnlein** [comment]
Thanks a lot for the updates!

**Pull request merged**